### PR TITLE
Auth pages

### PR DESCRIPTION
**Pull request by asvae**
see #955 

## Current status
**Work in progress**

image

## Leftovers and found issues
* Validation UX is rough on edges:
  * We're missing success marker for user registration password field. That's hard to do with current vuestic implementation (see https://github.com/epicmaxco/vuestic-ui/issues/3952).
  * messages for password doesn't disappear on for valid input.  
    image
  * Enter doesn't work for submit, you need to click with mouse.

## Ignored
* Errors with icons - tricky to do
  image
* Placeholders - I checked other refs and it's either placeholder or label
  image

### CODE REVIEW

**Comment by m0ksem** [comment]
Placeholder originally meant to be example@mail.com imo

File: src/pages/auth/Login.vue [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    <p class="text-base mb-4 leading-5">
+      New to Vuestic?
+      <router-link class="font-semibold text-primary" :to="{ name: 'signup' }">Sign up</router-link>
+    </p>
+    <va-input
+      v-model="formData.email"
+      class="mb-4"
+      type="email"
+      label="Email"
+      :rules="[(v) => !!v || 'Email field is required']"
```
Comment (m0ksem) [inline src/pages/auth/Login.vue:13]: Email validation?

File: src/pages/auth/Login.vue [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  const formData = reactive({
+    email: '',
+    password: '',
+    keepLoggedIn: false,
+  })
+
+  const submit = () => {
+    if (validate()) {
+      init({ message: "You've successfully logged in", color: 'success' })
+      push({ name: 'dashboard' })
```
Comment (m0ksem) [inline src/pages/auth/Login.vue:65]: Minor:
```suggestion
      router.push({ name: 'dashboard' })
```

File: src/pages/auth/VuesticLogo.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      :fill="colorComputed"
+    />
+    <path d="M64.7625 11.625V2.8125H62.25V0.375H70.0875V2.8125H67.5375V11.625H64.7625Z" :fill="colorComputed" />
+    <path d="M76.9875 11.625V0.375H79.7438V11.625H76.9875Z" :fill="colorComputed" />
+    <path
+      d="M87.2812 6C87.2812 2.64375 89.8125 0 93.375 0C95.5125 0 96.9938 0.80625 98.1938 2.0625L96.3 3.8625C95.5125 3.09375 94.5 2.60625 93.375 2.60625C91.3688 2.60625 90.075 4.10625 90.075 6C90.075 7.89375 91.3688 9.39375 93.375 9.39375C94.5 9.39375 95.5125 8.90625 96.3 8.1375L98.1375 9.9375C97.0125 11.1188 95.4562 12 93.375 12C89.8125 12 87.2812 9.35625 87.2812 6Z"
+      :fill="colorComputed"
+    />
+    <path
+      d="M106.553 8.58593C105.84 8.58593 105.308 8.3802 104.959 7.96874C104.616 7.55208 104.444 6.90885 104.444 6.03906V3.41406H105.788V6.19531C105.788 6.64843 105.847 6.96874 105.967 7.15624C106.092 7.33854 106.288 7.42968 106.553 7.42968C106.819 7.42968 107.014 7.33854 107.139 7.15624C107.269 6.96874 107.334 6.64843 107.334 6.19531V3.41406H108.631V6.03906C108.631 6.90885 108.462 7.55208 108.123 7.96874C107.785 8.3802 107.261 8.58593 106.553 8.58593Z"
```
Comment (m0ksem) [inline src/pages/auth/VuesticLogo.vue:32]: Must be Vuestic Admin, not Vuestic UI mb?

**Review by m0ksem — APPROVED** [review]

**Comment by m0ksem** [comment]
Would be nice to hide login page here:
image

Or I also saw somewhere login page was the first one in admin. It is something user might expect in template repo.

File: src/pages/auth/Login.vue (hunk 2 above)
Comment (asvae) [reply src/pages/auth/Login.vue:65]: Why?

**Comment by asvae** [comment]
> Would be nice to hide login page here: image
> 
> Or I also saw somewhere login page was the first one in admin. It is something user might expect in template repo.

We'll have this in other PRs, this one focuses only on page contents.

File: src/pages/auth/VuesticLogo.vue (hunk 3 above)
Comment (asvae) [reply src/pages/auth/VuesticLogo.vue:32]: Nice catch!

**Pull request merged**